relay-dynamic-config/src/metrics.rs

```diff
@@ -4,11 +4,27 @@ use std::collections::BTreeSet;
 
 use relay_base_schema::data_category::DataCategory;
 use relay_common::glob2::LazyGlob;
+use relay_common::glob3::GlobPatterns;
 use relay_protocol::RuleCondition;
 use serde::{Deserialize, Serialize};
 
 use crate::project::ProjectConfig;
 
+/// Configuration for metrics filtering.
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct Metrics {
+    /// Patterns of names of metrics that we want to filter.
```

Comment:
```suggestion
    /// The list of patterns which used to filter names of the received metrics.
```
Maybe something like this will sound better?

Reply:
hmm, isnt it a bit misleading, like we want to say we filter metrics based on their name, not that we filter their names?

how about `/// List of patterns for blocking metrics based on their name.` ?